galeracluster/source/documentation/inconsistency-voting.rst

```diff
@@ -0,0 +1,95 @@
+.. meta::
+   :title: Inconsistency Voting
+   :description:
+   :language: en-US
+   :keywords: galera cluster, inconsistency voting
+   :copyright: Codership Oy, 2014 - 2021. All Rights Reserved.
+
+
+.. container:: left-margin
+
+   .. container:: left-margin-top
+
+      :doc:`The Library <../index>`
+
+   .. container:: left-margin-content
+
+      .. cssclass:: here
+
+         - :doc:`Documentation <./index>`
+
+      - :doc:`Knowledge Base <../kb/index>`
+      - :doc:`Training <../training/index>`
+
+      .. cssclass:: sub-links
+
+         - :doc:`Training Courses <../training/courses/index>`
+         - :doc:`Tutorial Articles <../training/tutorials/index>`
+         - :doc:`Training Videos <../training/videos/index>`
+
+      - :doc:`FAQ <../faq>`
+      - :ref:`search`
+
+      Related Documents
+
+      - :ref:`gcs.vote_policy <gcs.vote_policy>`
+
+.. container:: top-links
+
+   - `Home <https://galeracluster.com>`_
+
+   .. cssclass:: here
+
+      - :doc:`Docs <./index>`
+
+   - :doc:`KB <../kb/index>`
+
+   .. cssclass:: nav-wider
+
+      - :doc:`Training <../training/index>`
+
+   - :doc:`FAQ <../faq>`
+
+
+.. cssclass:: library-document
+.. _`inconsistency-voting`:
+
+=================================
+Cluster Inconsistency Voting
+=================================
+
+.. index::
+   pair: Configuration; gcs.vote_policy
+
+
+Cluster inconsistency voting is a protocol for nodes to decide how the cluster reacts to problems in replication. Inconsistency voting helps, when one or several nodes have an issue to apply an incoming transactions. This can happen, for example, in the case of suspected inconsistency.
```

Comment:
please fix these sentences: "Inconsistency voting helps, when one or several nodes have an issue to apply an incoming transactions. This can happen, for example, in the case of suspected inconsistency."

Reply:
What is wrong with the current text? I don't know how to fix it.

Reply:
I edited it